I get a test failure for `python36Packages.root_numpy`

```
======================================================================
ERROR: root_numpy (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: root_numpy
Traceback (most recent call last):
  File "/nix/store/2lb6nn8ivk1alhckv43n7734lqwbw7h9-python3-3.6.2/lib/python3.6/unittest/loader.py", line 462, in _find_test_path
    package = self._get_module_from_name(name)
  File "/nix/store/2lb6nn8ivk1alhckv43n7734lqwbw7h9-python3-3.6.2/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/tmp/nix-build-python3.6-root_numpy-4.7.3.drv-0/root_numpy-4.7.3/root_numpy/__init__.py", line 21, in <module>
    ROOT_VERSION = root_version_active()
  File "/tmp/nix-build-python3.6-root_numpy-4.7.3.drv-0/root_numpy-4.7.3/root_numpy/setup_utils.py", line 111, in root_version_active
    import ROOT
  File "/nix/store/j2bzxvy45331h8319dxs5ziyhrkq8i2w-root-6.10.08/lib/ROOT.py", line 24, in <module>
    import cppyy
  File "/nix/store/j2bzxvy45331h8319dxs5ziyhrkq8i2w-root-6.10.08/lib/cppyy.py", line 61, in <module>
    import libPyROOT as _backend
ImportError: dynamic module does not define module export function (PyInit_libPyROOT)
----------------------------------------------------------------------
Ran 1 test in 0.000s
FAILED (errors=1)
```

--- a/pkgs/development/python-modules/root_numpy/default.nix
+++ b/pkgs/development/python-modules/root_numpy/default.nix
@@ -1,0 +1,23 @@
+{ lib, fetchPypi, isPy3k, buildPythonPackage, numpy, root }:
+
+buildPythonPackage rec {
+  pname = "root_numpy";
+  version = "4.7.3";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0vy6mqmkddfv46yc4hp43fvaisn3yw92ryaw031k841hhj73q0xg";
+  };
+
+  disabled = isPy3k; # blocked by #27649
+
+  propagatedBuildInputs = [ numpy root ];
+
+  meta = with lib; {
+    homepage = http://scikit-hep.org/root_numpy;
+    license = licenses.bsd3;
+    description = "The interface between ROOT and NumPy";
+    maintainers = with maintainers; [ veprbl ];
+  };
+}

--- a/pkgs/development/python-modules/rootpy/default.nix
+++ b/pkgs/development/python-modules/rootpy/default.nix
@@ -1,0 +1,23 @@
+{ lib, fetchPypi, isPy3k, buildPythonPackage, numpy, matplotlib, root, root_numpy, tables }:
+
+buildPythonPackage rec {
+  pname = "rootpy";
+  version = "1.0.1";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0zp2bh87l3f0shiqslbvfmavfdj8m80y8fsrz8rsi5pzqj7zr1bx";
+  };
+
+  disabled = isPy3k;
+
+  propagatedBuildInputs = [ matplotlib numpy root root_numpy tables ];
+
+  meta = with lib; {
+    homepage = http://www.rootpy.org;
+    license = licenses.bsd3;
+    description = "Pythonic interface to the ROOT framework";
+    maintainers = with maintainers; [ veprbl ];
+  };
+}

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -18919,6 +18919,8 @@ in {
     };
   };
 
+  root_numpy = callPackage ../development/python-modules/root_numpy { };
+
   rootpy = buildPythonPackage rec {
     version = "0.9.0";
     name = "rootpy-${version}";

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -18921,25 +18921,7 @@ in {
 
   root_numpy = callPackage ../development/python-modules/root_numpy { };
 
-  rootpy = buildPythonPackage rec {
-    version = "0.9.0";
-    name = "rootpy-${version}";
-
-    src = pkgs.fetchurl {
-      url = "mirror://pypi/r/rootpy/${name}.tar.gz";
-      sha256 = "04alx6afiyahhv816f6zpwnm0sx2jxgqpgqcn6kdw0wnpc9625cr";
-    };
-
-    disabled = isPy3k;
-
-    propagatedBuildInputs = with self; [ pkgs.root numpy matplotlib ];
-
-    meta = {
-      homepage = "http://www.rootpy.org";
-      license = licenses.gpl3;
-      description = "Pythonic interface to the ROOT framework";
-    };
-  };
+  rootpy = callPackage ../development/python-modules/rootpy { };
 
   rope = buildPythonPackage rec {
     version = "0.10.2";

